tests/unit_test.py

```diff
@@ -917,12 +917,13 @@ def get_ray_obj(group, version, namespace, plural, cls=None):
                 "apiVersion": "ray.io/v1",
                 "kind": "RayCluster",
                 "metadata": {
-                    "creationTimestamp": "2023-02-22T16:26:07Z",
+                    "creationTimestamp": "2024-03-05T09:55:37Z",
                     "generation": 1,
                     "labels": {
-                        "workload.codeflare.dev/appwrapper": "quicktest",
```

Comment:
Does it matter which of these we use here? Should be whatever is most up to date I think

Reply:
This was taken straight from an existing RC.
I am not sure why the `appwrapper.mcad.ibm.com` label is there I thought it was removed. 

@ChristianZaccaria iirc you had worked on removing this label?

Reply:
Unfortunately that PR with many others didn't get merged: https://github.com/project-codeflare/multi-cluster-app-dispatcher/pull/652/files